tests/test_module_loader.py

```diff
@@ -78,7 +86,7 @@ def test_v1_and_v2_module_installed_simultaneously(
        * The V2 module is loaded and not the V1 module.
     """
     # Work around caching problem in venv
-    feature_compiler_cache.set("False")
+    # feature_compiler_cache.set("False")
```

Comment:
Same as above, no difference here when it's set to `False` or `True`.

Reply:
remove!